### PR TITLE
Add rollbar

### PR DESCRIPTION
**Pull request by ajm188**
Note that we will have to update the deploy environment to include ROLLBAR_ACCESS_TOKEN

### CODE REVIEW

**Comment by matthewbentley** [comment]
Resolve conflicts and merge.  The access token is in place on the server

**Comment by ajm188** [comment]
Rebased and fixed. Waiting on travis before merging.

**Comment by ajm188** [comment]
Looks like I messed up with rebase - I'll fix this next week

File: Gemfile.lock [hunk 1]
```diff
@@ -238,13 +238,22 @@ GEM
...
+    rollbar (2.15.0)
+      multi_json
     rspec (3.6.0)
       rspec-core (~> 3.6.0)
       rspec-expectations (~> 3.6.0)
       rspec-mocks (~> 3.6.0)
     rspec-core (3.6.0)
       rspec-support (~> 3.6.0)
     rspec-expectations (3.6.0)
+    rspec (3.5.0)
```
Comment (ajm188) [inline Gemfile.lock]: lol i see the issue.
me--

**Pull request merged**